.gitignore

```diff
@@ -1,2 +1,4 @@
 .DS_Store
 node_modules
+crossfilter.js
+crossfilter.min.js
```

Comment:
If you add files to gitignore, you need to manually remove them from the repo, otherwise it just ignores any further changes you make to the file.

Reply:
files remover, thanks.